### ISSUE TITLE
PagePiling.js not working even using repo example code 

### ISSUE THREAD

**Issue by ricardozea** [alvarotrigo/pagePiling.js#103]
Hello,

PagePiling.js isn't working even using the example code from the repo. Check this demo out: http://codepen.io/ricardozea/pen/a38ac610b5bf994216d3804ee8e432d3

All sections are visible in the beginning.

Am I doing something wrong? Or the script really not working as intended?

Thanks

**Comment by alvarotrigo** [comment]
> All sections are visible in the beginning.

Of course, you need to add a background to the section so the ones behind won't be visible.

**Comment by ricardozea** [comment]
Oh ok, that was easy. 

I know it sounds obvious to you, you're the creator, but it would be nice if you could clarify this somewhere in your documentation. I see you have something in the [Options](https://github.com/alvarotrigo/pagePiling.js#options) section, but it's on the JavaScript side, which is fine. The thing is that designers will very likely add backgrounds via CSS or in the markup if they're using video.

Thanks for your fast reply and for this awesome script.

+1

**Comment by alvarotrigo** [comment]
Labeled as a documentation enhancement.
